kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart

```diff
@@ -229,6 +229,20 @@ class _ResponseCardState extends State<_ResponseCard> {
   /// The [Uri] that the [_request] requested.
   Uri get _requestUri => _request.uri;
 
+  String? get _requestHeadersBuilder {
+    final Map<String, List<String>> map = _request.headers.map(
+      (key, value) => MapEntry(
+        key,
+        value is Iterable ? value.map((v) => v.toString()).toList() : [value],
+      ),
+    );
+    final Headers headers = Headers.fromMap(map);
+    if (!headers.isEmpty) {
+      return '\n$headers';
```

Comment:
Do you think `_requestHeadersBuilder` should return a pure header `String` without the `\n`?

Reply:
And put it to the Widget tree